uint8 ACTIVATE=4
uint8 ACTIVATE_MARKED=5
uint8 DEACTIVATE=6
uint8 DEACTIVATE_MARKED=7
uint8 SWITCH_TO_WORKPIECE=8
uint8 SWITCH_TO_CONVEYOR=9
uint8 SWITCH_TO_SLIDE=10
uint8 SWITCH_OFF_DETECTION=11

uint8 command
---
bool success
string message
